app/main.py

```diff
@@ -1,8 +1,21 @@
 class Person:
-    # write your code here
-    pass
+    people = dict()
+
+    def __init__(self,
+                 name: str,
+                 age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = [Person(person["name"], person["age"])
+                   for person in people]
+    instance = Person.people
```

Comment:
There is no need to cross-assign a class attribute to another variable, use the class attribute directly. In python, it is uncommon to create a variable to store the reference to another variable without any additional modifications.